ui/app/bedmanagement/views/wardListPrint.html

```diff
@@ -0,0 +1,109 @@
+<style>
+    @page {
+        size: A4 landscape;
+    }
+
+    body {
+        font-family: sans-serif;
+        font-size: 10px !important;
```

Comment:
Can we avoid using important?

Reply:
Done

Reply:
You have removed 'important' without any other change, does it still work?

Reply:
Yes. I checked. I found no change